src/metabolic_ninja/worker/helpers.py

```diff
@@ -174,3 +174,33 @@ def identify_exotic_cofactors(pathway, model, threshold=1e-07):
         # Add new reactions to exploration queue.
         rxn_queue.extend((target.reactions & heterologous_reactions) - seen)
     return exotic_cofactors
+
+
+def manipulation_helper(target):
+    """Convert a FluxModulationTarget to a dictionary."""
+    result = {"id": target.id, "value": target.fold_change}
```

Comment:
At least in this PR the flux inversion is missing. I think you patched that already.

Reply:
Can confirm it's patched in the merge :+1: